cmake/Config.cmake.in

```diff
@@ -0,0 +1,18 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(@ARABICA_HAVE_BOOST@)
+    find_dependency(Boost)
+endif()
+
+if(@ARABICA_XML_BACKEND@ STREQUAL USE_EXPAT)
```

Comment:
It should be `@USE_EXPAT@`. Also, it will be more user-friendly to put everything in quotes. If both variables are empty, the generated line will be `if( STREQUAL )`. Not sure if it will work correctly, but it will be much easier to understand if you generate `if("" STREQUAL "")` instead.

Reply:
Sure, I will wrap everything in quotes.